support-frontend/assets/components/subscriptionCheckouts/endSummary/endSummarySelector.js

```diff
@@ -1,14 +1,16 @@
 // @flow
 
 import { getProductPrice } from 'helpers/productPrice/productPrices';
+import { getPaperProductPrice } from 'helpers/productPrice/paperProductPrices';
 
 import { type Option } from 'helpers/types/option';
 import type { ProductPrice } from 'helpers/productPrice/productPrices';
 import type { CheckoutState } from 'helpers/subscriptionsForms/subscriptionCheckoutReducer';
 import { getBillingDescription } from 'helpers/productPrice/priceDescriptionsDigital';
+import { getPaymentStartDate } from 'pages/paper-subscription-checkout/helpers/options';
 
 export type EndSummaryProps = {
-  priceDescription: string,
+  priceDescription: Option<?string>,
```

Comment:
What does this type annotation mean? Shouldn't it just be `Option<string>`? or `priceDescription? : Option<string>`?

Reply:
Yes, you're right! A slip of the key 👍